### PR TITLE
Coverage - looks good

### PR DESCRIPTION
**Pull request by raghu1978**

### CODE REVIEW

File: app/services/CustomsCacheService.scala [hunk 1]
```diff
@@ -27,6 +27,8 @@ import uk.gov.hmrc.wco.dec.inventorylinking.movement.request.InventoryLinkingMov
 
 import scala.concurrent.{ExecutionContext, Future}
 
+// $COVERAGE-OFF$Trivial and never going to be called by a test that uses it's own object implementation
```
[the remark this answers is not in the document]
Comment (raghu1978) [reply app/services/CustomsCacheService.scala]: will update comment that doesn't need the comment

File: app/services/CustomsCacheService.scala [hunk 2]
```diff
@@ -27,6 +27,8 @@ import uk.gov.hmrc.wco.dec.inventorylinking.movement.request.InventoryLinkingMov
 
 import scala.concurrent.{ExecutionContext, Future}
 
+// $COVERAGE-OFF$Trivial
```
Comment (PatrykRudnicki) [inline app/services/CustomsCacheService.scala]: Trivial is necessary?

Comment (raghu1978) [reply app/services/CustomsCacheService.scala]: yeah just to understand

Comment (PatrykRudnicki) [reply app/services/CustomsCacheService.scala]: ok

**Pull request merged**